src/searchbar/SearchBar-ios.js

```diff
@@ -41,6 +41,17 @@ class SearchBar extends Component {
     };
   }
 
+  componentDidMount() {
```

Comment:
@xavier-villelegier what do you think about this?

Reply:
Mmh I think this should work, but I personally would use the `onLayout` event for this, to have only one `setState` for all measures

Have a look at [line 182](https://github.com/react-native-training/react-native-elements/pull/1533/files#diff-348f551797bae8f23921aab3699399eeR182)

Reply:
Yeah... I've tried that, but `onLayout is invoked on mount and layout changes', so it won't be calculated once but multiple times (whenever user will focus or blur input) :(

Reply:
Ah, that makes sense… Maybe let's remove the `onLayout` then ? They both override the `cancelButtonWidth`

Reply:
I mean we could just check the value of cancelButtonWidth? If it’s already set then don’t update the state and set it again. 

Also I couldn’t seem to get the tests running with measure.